crates/sui-indexer-alt-jsonrpc/src/config.rs

```diff
@@ -64,10 +68,18 @@ pub struct RpcLayer {
     /// Configuring limits for the package resolver.
     pub package_resolver: PackageResolverLayer,
 
+    /// Threshold for logging slow requests in seconds. Default: 60s
+    #[serde(with = "serde_with::As::<serde_with::DurationSeconds<u64>>")]
+    pub slow_request_threshold: Duration,
```

Comment:
We try to keep time-based configs consistent, namely: They are always in milliseconds, and prefixed with `_ms`. Also, no need to quote a default -- this type implements `DefaultConfig`, so the value of `RpcLayer::default().slow_request_threshold_ms` will be used here.

```suggestion
    /// Threshold for logging slow requests in milliseconds.
    pub slow_request_threshold_ms: usize,
```

You can do the conversion into a `Duration` when you finish the `Layer` to get the `Config`.